### PR TITLE
Nada/87301/trade parameter UI

### PR DESCRIPTION
**Pull request by nada-deriv**

### CODE REVIEW

File: src/routes/trade/trade.module.scss [hunk 1]
```diff
@@ -78,68 +108,93 @@
     }
   }
   &__buy {
-    background-color: #3d9494;
+    background-color: #4ab5b3;
```
Comment (likhith-deriv) [inline src/routes/trade/trade.module.scss]: Can u move these colors to theme file and use the variables. Also please check if they need any change in dark and light mode

Comment (nada-deriv) [reply src/routes/trade/trade.module.scss]: updated.

**Review by likhith-deriv — APPROVED** [review]

File: src/components/dropdown-component.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  document.addEventListener("click", (event) => {
+    if (
+      !document
+        .getElementById(`dropdown-wrapper-${props.el_id}`)
+        ?.contains(event.target)
+    ) {
+      setShowDropdownList(false);
+    }
+  });
```
Comment (farrah-deriv) [inline src/components/dropdown-component.jsx:26]: what is this for?

File: src/routes/trade/trade.module.scss [hunk 3]
```diff
@@ -51,8 +71,14 @@
...
   }
 
+  &__button {
+    background-color: var(--section-background-1);
+    color: var(--text-main);
+    font-weight: bold;
+  }
   &--selected {
-    background-color: #939797;
+    background-color: #d7dadb;
```
Comment (farrah-deriv) [inline src/routes/trade/trade.module.scss]: can we use the same color in `theme.scss`?

File: src/components/dropdown-component.jsx (hunk 2 above)
Comment (nada-deriv) [reply src/components/dropdown-component.jsx:26]: closing the dropdown on backdrop click

File: src/routes/trade/trade.module.scss (hunk 3 above)
Comment (nada-deriv) [reply src/routes/trade/trade.module.scss]: updated

**Review by farrah-deriv — APPROVED** [review]

**Pull request merged**